src/cli/helpers.ts

```diff
@@ -216,6 +217,34 @@ function emitTestCaseHooks(
     })
 }
 
+function emitTestStepHooks(
+  supportCodeLibrary: ISupportCodeLibrary,
+  eventBroadcaster: EventEmitter
+): void {
+  ;[]
+    .concat(
+      supportCodeLibrary.beforeTestStepHookDefinitions,
```

Comment:
Would it not make more sense to emit the before step hooks before the testStepStarted messages are emitted, and the after step hooks after?

Reply:
@davidjgoss @aslakhellesoy Is my assumption about what you mean correct? I can work on this today as soon as I get a confirmation (or correction if my understanding is wrong).

Reply:
Removed function per Charlie's Comment about it.